var/spack/repos/builtin/packages/py-pyprecice/package.py

```diff
@@ -39,12 +46,13 @@ class PyPyprecice(PythonPackage):
 
     depends_on("python@3:", type=("build", "run"))
     depends_on("py-setuptools", type="build")
+    # py-wheel dependency is specified in pyproject.toml of pyprecice
     depends_on("py-wheel", type="build")
```

Comment:
Even though it's specified in `pyproject.toml`, I don't think it's needed to install, it's only needed to build wheels to upload to PyPI.